solr/bin/solr

```diff
@@ -2312,13 +2341,13 @@ function start_solr() {
 	echo ""
     fi
     # no lsof on cygwin though
-    if lsof -v 2>&1 | grep -q revision; then
+    if [ -n "$(get_port_tool)" ]; then
```

Comment:
rather than seeking the port tool a second time (calling `get_port_tool()` from within the `check_port_in_use()` function), would it be just as clean/easy to assign the output of `get_port_tool` to a variable, check that variable for `[ -n "$PORT_TOOL" ]`, and pass `$PORT_TOOL` as an arg to `check_port_in_use()`?

Fine as-is, but seems there'd be very little downside to this change?